squidb/src/com/yahoo/squidb/data/ValuesStorage.java

```diff
@@ -179,6 +179,18 @@ public void put(@Nonnull String key, @Nullable Object value, boolean errorOnFail
     }
 
     @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        for (String key : keySet()) {
```

Comment:
Oh, one year doing only little java and I already forgot about this, shame on me!